<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LOGIN</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
color:rgb(255, 0, 0);

}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>600</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(62, 62, 62);
border-radius:15px;

border:2px solid rgb(255, 212, 0)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>20</y>
            <width>201</width>
            <height>151</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:3px solid rgb(255, 212, 0);
border-radius:75px;
background-color: rgb(141, 141, 141);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../a.qrc">:/newPrefix/icones/logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QFrame" name="frame_4">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>180</y>
            <width>361</width>
            <height>281</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:0px</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
color:rgb(141, 141, 141);
border:2px solid rgb(255, 212, 0);
border-radius:20px
}
QLineEdit:hover{
border:3px solid rgb(220, 180, 0)
}</string>
             </property>
             <property name="maxLength">
              <number>30</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>USERNAME</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
color:rgb(141, 141, 141);
border:2px solid rgb(255, 212, 0);
border-radius:20px
}
QLineEdit:hover{
border:3px solid rgb(220, 180, 0)
}
</string>
             </property>
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>PASSWORD</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QCheckBox{
color:rgb(255, 255, 255);
}

QCheckBox:indicator{
width:15px;
height:15px;
border:3px solid rgb(141, 141, 141);
border-radius:10px
}
QCheckBox:indicator:checked{
background-color: rgb(255, 255, 0);
border:3px solid rgb(255, 212, 0)
}</string>
             </property>
             <property name="text">
              <string>SALVAR USUARIO</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>140</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI Variable Small Semibol</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
               <kerning>false</kerning>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:0px;
color:rgb(85, 170, 255)
}
QPushButton:hover{
font: 63 12pt &quot;Segoe UI Variable Small Semibol&quot;;
}</string>
             </property>
             <property name="text">
              <string>ESQUECI A SENHA</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frame_5">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>460</y>
            <width>361</width>
            <height>111</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:0px</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
           <property name="spacing">
            <number>60</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:2px solid rgb(255, 212, 0);
border-radius:25px;
font: 63 12pt &quot;Yu Gothic UI Semibold&quot;;
	background-color: rgb(141, 141, 141)
}
QPushButton:hover{
border:4px solid rgb(255, 220, 0);
font: 63 15pt &quot;Yu Gothic UI Semibold&quot;;

}</string>
             </property>
             <property name="text">
              <string>CADASTRAR</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>60</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
border:2px solid rgb(255, 212, 0);
border-radius:25px;
font: 63 12pt &quot;Yu Gothic UI Semibold&quot;;
	background-color: rgb(141, 141, 141)
}
QPushButton:hover{
border:4px solid rgb(255, 220, 0);
font: 63 15pt &quot;Yu Gothic UI Semibold&quot;;

}</string>
             </property>
             <property name="text">
              <string>ENTRAR</string>
             </property>
             <property name="shortcut">
              <string>Return</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frame_6">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>480</y>
            <width>70</width>
            <height>70</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border:0px</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">border:0px</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../a.qrc"/>
 </resources>
 <connections/>
</ui>
